Desktop/property-pro-app/src/components/middlepage/middlepage.css

```diff
@@ -0,0 +1,45 @@
+.middlepage {
+  height: 55vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.change {
+  font-size: x-large;
+  color: whitesmoke;
+  font-style: oblique;
+  margin-top: 30px;
+  /* z-index: 999; */
+}
+
+.search-input-field {
+  width: 100%;
+  padding: 10px 10px;
+  margin-top: 5px;
+  /* z-index: 999; */
+}
+#search {
+  z-index: 999;
+}
+
+@media screen and (min-width: 768px) {
+  .middlepage {
+    height: 85vh;
+    width: 100%;
+    background: url("/public/assets/image33.jpg");
+    background-position: 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+  }
+  .middlepage::after {
+    content: "";
```

Comment:
What are we using this for? If nothing lets take it out and if there is add it up.